packetbeat/protos/nfs/nfs4.go

```diff
@@ -165,20 +165,37 @@ func (nfs *Nfs) eatData(op int, xdr *Xdr) {
 	}
 }
 
-func (nfs *Nfs) getV4Opcode(xdr *Xdr) string {
+// Find main operation in a compound call.
+//
+//  Compound request group multiple nfs operations into a single request. Nevertheless, all compound requests are
+// triggered by  end-user activity, line 'ls', 'open', 'stat' and IO calls. Depending on which operations are combined
+//  tougher the main operation can be different. For example, in compound:
+//
+// PUTFH + READDIR + GETATTR
+//
+// READDIR is the main operation. while in
+//
+// PUTFH + GETATTR
+//
+// GETATTR is the main operation.
+//
+//  This  function tries to guess main purpose of the request. If not found, then the last operation is used.
```

Comment:
Very nice explanation. Thanks!!!

some minor details:
- I don't see the caller of getV4Opcode being changed. Does it compile?
- spacing on line 170 and 172 seems to be a little off
- godoc convention kinda mandates putting the function and short description first (before going into detail):
```
findV4MainOpcode finds the main operation in a compound call. If no main operation can be found, the last operation in compound call is returned.

Compount requests group ...
```

You can remove the last line `This function tries...`, as this information should be present at beginning of comment (see my sample).